web/client/test-resources/drawsupport/features.js

```diff
@@ -52,5 +52,498 @@ module.exports = {
         },
         id: "multipolygon",
         properties: {}
+    },
+    lineFeature: {
+        type: "Feature",
+        geometry: {
+            type: "LineString",
+            coordinates: [[0, 1], [2, 4]]
+        },
+        properties: {}
+    },
+    lineFeature3: {
+        type: "Feature",
+        geometry: {
+            type: "LineString",
+            coordinates: [[13.0, 43.0], [13.0, 40.0], [11.0, 41.0]]
+        },
+        properties: {}
+    },
+    lineFeature2: {
+        type: "Feature",
+        geometry: {
+            type: "LineString",
+            coordinates: [[1, 1], [55, 5]]
+        },
+        properties: {}
+    },
+    geomCollFeature: {
+        type: "Feature",
+        geometry: {
+            type: "GeometryCollection",
+            geometries: [{
+                type: "LineString",
+                coordinates: [[0, 1], [2, 4]]
+            }]
+        },
+        properties: {}
+    },
+    invalidLineFeature: {
+        type: "Feature",
+        geometry: {
+            type: "LineString",
+            coordinates: [[1, 1], [55, 5], ["", 5]]
+        }
+    },
+    polyFeature: {
+        type: "Feature",
+        geometry: {
+            type: "Polygon",
+            coordinates: [[[0, 1], [0, 5], [2, 1], [0, 1]]]
+        },
+        properties: {}
+    },
+    invalidPolyFeature: {
+        type: "Feature",
+        geometry: {
+            type: "Polygon",
+            coordinates: [[[0, 1], [0, 5], [10, 5], [2, ""], [0, 1]]]
+        },
+        properties: {}
+    },
+    circle: {
+        type: 'FeatureCollection',
+        id: '36835090-23ad-11e8-9839-9bab136db9a3',
+        newFeature: true,
+        properties: {
+            id: '36835090-23ad-11e8-9839-9bab136db9a3',
+            circles: [0],
+            isCircle: true
+        },
+        style: {
+            type: 'Circle',
+            Circle: {
+                color: '#ffcc33',
+                opacity: 1,
+                weight: 3,
+                fillColor: '#ffffff',
+                fillOpacity: 0.2,
+                radius: 10
+            }
+        },
+        features: [{
+            type: 'Feature',
+            geometry: {
+                type: 'Polygon',
+                coordinates: [
+                [
+                  [
+                    -7.066692092065635,
+                    47.17477833929903
+                  ],
+                  [
```

Comment:
I think 4 points are enough (the first and last of course should remain)

Reply:
this is a circle transformed into a polygon with 100 points and it is needed.
maybe we can add a function that generates random coords or reuse [transformLineToArcs](https://github.com/geosolutions-it/MapStore2/blob/c125_annotations/web/client/utils/CoordinatesUtils.js#L554)